### PR TITLE
Getting logits to the client code

### PR DESCRIPTION
**Pull request by adeepH**
Follow up to #2075, and #2339, to return the model outputs to the user.

### CODE REVIEW

**Comment by ftyers** [comment]
@reuben can we get approval to run the workflows?

**Comment by reuben** [comment]
@ftyers sure. I've approved this push and gave @adeepH write access so future pushes should run automatically.

**Comment by adeepH** [comment]
@reuben  the commits did pass the Lint tests but I am not sure what went wrong in the tests that failed. Do you happen to know what went wrong?

**Comment by reuben** [comment]
Seems to be a couple of independent issues with the various platforms.

**Comment by ftyers** [comment]
@reuben thoughts on how to resolve them? Some of them don't seem to be issues with this PR, but rather general issues, no?

**Comment by adeepH** [comment]
As skeilnet suggested, I've created a new branch (logits) and I have raised another pull request #2352. 

I will be closing this PR.
